Packs/DarktraceASM/Integrations/DarktraceASMRisk/DarktraceASMRisk.yml

```
@@ -0,0 +1,302 @@
+commonfields:
+  id: Darktrace ASM
+  version: -1
+name: Darktrace ASM
+display: Darktrace ASM
+category: Network Security
+description: 'This pack includes configurations to combine the world-class threat
+  detection of Darktrace with the synchrony and automation abilities of XSOAR, allowing
+  security teams to monitor their attack surface for risks, high-impact vulnerabilities
+  and external threats.\nTo configure the connection to your Darktrace Attack Surface 
+  Management instance, you will provide:\n- Server URL of Darktrace ASM instance (ex: 
+  darktrace.yourcompany.com) and any necessary proxy information\n- The API Token from the Darktrace ASM instance'
+detaileddescription: "### Community Contributed Integration\n#### Integration Author:
+  Darktrace\nSupport and maintenance for this integration are provided by the author.
+  Please use the following contact details:\n- Email: integrationsupport@darktrace.com\n-
+  URL: https://customerportal.darktrace.com\n***\n## Darktrace ASM \nTo configure
+  the connection to your Darktrace instance, you will provide:\n- Server URL of Darktrace
+  ASM instance (ex: example-asm.darktrace.com) and any necessary proxy information\n-
+  Darktrace ASM API token\n\n---
+  \n[View Integration Documentation](https://xsoar.pan.dev/docs/reference/integrations/darktrace-admin)"
+fromversion: 6.6.0
+defaultmapperin: DarktraceASM Risk Incoming Mapper
```

```suggestion
defaultmapperin: Darktrace ASM Risk Mapper
```

There was a typo there

---

fixed this typo